app/controllers/sessions_controller.rb

```diff
@@ -14,5 +14,7 @@ def create
   end
 
```

Comment:
Anytime you use the words 'Additionally' and 'Also' in a commit message question if the commit is a cohesive, atomic commit. 

A commit should be the smallest amount of functional code that you can implement